x/evm/types/tx.pb.go

```diff
@@ -197,7 +197,7 @@ func (m *AccessListTx) XXX_DiscardUnknown() {
 
 var xxx_messageInfo_AccessListTx proto.InternalMessageInfo
 
-// DynamicFeeTx is the data of EIP-1559 dinamic fee transactions.
+// DynamicFeeTx is the data of EIP-1559 dynamic fee transactions.
```

Comment:
Can you change this in the `tx.proto` which is where it's coming from and regenerate the protos please.

Reply:
```
protoc-c --c_out=. tx.proto
cosmos/msg/v1/msg.proto: File not found.
cosmos_proto/cosmos.proto: File not found.
ethermint/evm/v1/evm.proto: File not found.
gogoproto/gogo.proto: File not found.
google/api/annotations.proto: File not found.
google/protobuf/any.proto: File not found.
tx.proto:6:1: Import "cosmos/msg/v1/msg.proto" was not found or had errors.
```

not sure how to re-generate` tx.pb.go `or I should just modify dynamic in` tx.proto`?

Reply:
yes modify the typo in `tx.proto` and then you can run `make proto-all` command from the root folder.

Reply:
1. at first I revert dynamic-> dinamic in [x/evm/types/tx.pb.go](https://github.com/evmos/evmos/pull/2199/files/adfc44b087629901c3edb6ca3364222d7f997cac#diff-7ca01794cb14e52d5aee1d0d8e04802d97b7488cd8cbba3c45c8f809584b43b6)

2. then run make proto-all,just modify dinamic->dynamic, else have no change. so seems don't need modify 
```
make proto-all
Formatting Protobuf files
/usr/bin/docker run --rm -v /workspaces/evmos:/workspace --workdir /workspace --user 0 ghcr.io/cosmos/proto-builder:0.11.6 find ./ -name *.proto -exec clang-format -i {} \;
Linting Protobuf files
Generating Protobuf files
/usr/bin/docker run --rm -v /workspaces/evmos:/workspace --workdir /workspace --user 0 ghcr.io/cosmos/proto-builder:0.11.6 sh ./scripts/protocgen.sh
Generating gogo proto code
```

Reply:
Ahh yes you modified it all manually nice, didn't see it before 👍🏻